pkg/test/resourcefixture/contexts/bigqueryconnectionconnection_context.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package contexts
+
+func init() {
+	resourceContextMap["cloudresourceconnection"] = ResourceContext{
+		ResourceKind: "BigQueryConnectionConnection",
+		SkipUpdate:   true,
```

Comment:
Is this resource not updatable at all?

Reply:
For connection type "cloudResource", yes update operation is not supported.